db/schema.rb

```diff
@@ -10,8 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_07_18_164661) do
-  create_table "author_identities", force: :cascade do |t|
+ActiveRecord::Schema[7.2].define(version: 2022_07_18_164661) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "author_identities", id: :serial, force: :cascade do |t|
```

Comment:
You'll see `id: :serial` added to each table creation, which is what rails adds for PG DB migrations.